sdk/identity/Azure.Identity/src/ClientCertificateCredential.cs

```diff
@@ -205,7 +205,7 @@ internal interface IX509Certificate2Provider
         /// <summary>
         /// X509Certificate2FromObjectProvider provides an X509Certificate2 from an existing instance.
         /// </summary>
-        private class X509Certificate2FromObjectProvider : IX509Certificate2Provider
+        internal class X509Certificate2FromObjectProvider : IX509Certificate2Provider
```

Comment:
nit: should we move these nested types out of `ClientCertificateCredential` into a standalone types since they are used in other classes now? Or we can add an issue to refactor when we have time.